Core/Utilities/SiteSEMapping.py

```diff
@@ -8,10 +8,86 @@
 
 __RCSID__ = "$Id$"
 
-from DIRAC import S_OK
+import errno
+
+from DIRAC import gLogger, S_OK, S_ERROR
 from DIRAC.DataManagementSystem.Utilities.DMSHelpers import DMSHelpers, siteGridName
 
 
+def getSEParameters(seName, plugins=None):
+  """ get all the SE parameters in a list
+
+      :param str seName: name of the Storage Element
+      :param list plugins: if provided, restrict to a certain list of plugins
+
+      :return: S_OK() or S_ERROR
+  """
+  # This import is here to avoid circular imports
+  from DIRAC.Resources.Storage.StorageElement import StorageElement
+  se = StorageElement(seName, hideExceptions=True)
+
+  seParameters = S_ERROR(errno.ENODATA, 'No SE parameters obtained')
+  pluginsList = se.getPlugins()
+  if not pluginsList['OK']:
+    gLogger.warn(pluginsList['Message'], "SE: %s" % seName)
+    return pluginsList
+  if plugins:
+    pluginsSet = set(pluginsList['Value']).intersection(set(plugins))
+  else:
+    pluginsSet = set(pluginsList['Value'])
+
+  seParametersList = []
+  for plugin in pluginsSet:
+    seParameters = se.getStorageParameters(plugin)
+    if seParameters['OK']:
+      seParametersList.append(seParameters['Value'])
+
+  return S_OK(seParametersList)
```

Comment:
I am not sure this works with Echo for example, where the two plugins are called the same.

Reply:
```
In [7]: getSEParameters('RAL-DST')
Out[7]: 
{'OK': True,
 'Value': [{'Endpoint': 'root://xrootd.echo.stfc.ac.uk',
   'Host': 'xrootd.echo.stfc.ac.uk',
   'InputProtocols': ['root', 'file'],
   'OutputProtocols': ['root'],
   'Path': 'lhcb:prod',
   'PluginName': 'Echo',
   'Port': '',
   'Protocol': 'root',
   'SpaceToken': '',
   'StorageName': 'RAL-ECHO',
   'URLBase': 'root://xrootd.echo.stfc.ac.uk/lhcb:prod/',
   'WSUrl': ''}]}
```